### PR TITLE
Fix Cookies policy,  Remove IconButton on User not logged in App Bar

### LINKED ISSUE #2257: Cookies policy is appearing everytime.
**Actual Behaviour**

Cookies policy is appearing at the bottom, every time the webpage loads.

**Expected Behaviour**

It should be shown to the user until the user accepts the policy.

**Steps to reproduce it**

1. Visit https://susi.ai/.
2. Accept the policy.
3. Reload the page.

**Screenshots of the issue**

image

**Would you like to work on the issue?**

No, anyone can take this.

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2257, #2259

Changes: 
- [x] Fix small button present in StaticAppBar, TopAppBar when user not logged in.
- [x] Fix Cookies policy appearing at the bottom on page refresh, every time the webpage loads.

Demo Link: https://pr-2260-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -110,8 +112,9 @@ class App extends Component {
       renderFooterPagesList.includes(pathname) ? (
         <Footer />
       ) : null;
-    const renderCookiePolicy = showCookiePolicy ? <CookiePolicy /> : null;
-    const renderDialog = isModalOpen ? <DialogSection /> : null;
+    const renderCookiePolicy =
+      showCookiePolicy === true ? <CookiePolicy /> : null;
```
Comment (akshatnitd) [inline src/App.js]: Why is `=== true` required, if showCookiePolicy is already boolean?

File: src/App.js [hunk 2]
```diff
@@ -110,8 +112,9 @@ class App extends Component {
...
+    const renderDialog = isModalOpen || !visited ? <DialogSection /> : null;
```
Comment (akshatnitd) [inline src/App.js]: Why have you used `|| !visited`? Won't it open DialogSection if `visited` is false? If you intend to open the Dialog, then where is the type of Dialog defined?

Comment (fragm3) [reply src/App.js]: The video tour dialog will not open if we don't use this condition, it opens when visited is set to false in `DialogSection`.

File: src/App.js (hunk 1 above)
Comment (fragm3) [reply src/App.js]: It is not working with
`const renderCookiePolicy = showCookiePolicy ? <CookiePolicy /> : null`

**Pull request merged**